packages/iris-grid/src/IrisGridTheme.module.scss

```diff
@@ -12,92 +12,90 @@ $header-separator-color: $gray-900;
 $header-height: 30px;
 
 :export {
-  grid-bg: $gray-900;
+  grid-bg: var(--dh-color-grid-bg);
```

Comment:
Many of the variables above are no longer used (e.g. `$selection-color`).
Also the new selection color doesn't match the old color; I think that may be intentional? Confirm with @dsmmcken , but that's one reason the goto row tests are failing.

Reply:
I deleted the unused vars. Will check with Don on the selection color diff.
<img width="313" alt="image" src="https://github.com/deephaven/web-client-ui/assets/1900643/402e98b9-65e7-4b1f-bb26-1ecb4c2710df">